x-pack/plugins/spaces/server/lib/request_interceptors/on_post_auth_interceptor.ts

```diff
@@ -6,31 +6,30 @@
 import { Logger, CoreSetup } from 'src/core/server';
 import { Space } from '../../../common/model/space';
 import { wrapError } from '../errors';
-import { XPackMainPlugin } from '../../../../xpack_main/xpack_main';
-import { SpacesServiceSetup } from '../../new_platform/spaces_service/spaces_service';
-import { LegacyAPI } from '../../new_platform/plugin';
+import { SpacesServiceSetup } from '../../spaces_service/spaces_service';
+import { LegacyAPI, PluginsSetup } from '../../plugin';
 import { getSpaceSelectorUrl } from '../get_space_selector_url';
 import { DEFAULT_SPACE_ID, ENTER_SPACE_PATH } from '../../../common/constants';
-import { addSpaceIdToPath } from '../../../common';
+import { addSpaceIdToPath } from '../../../common/lib/spaces_url_parser';
```

Comment:
__nit:__ I guess you could just leave `../../../common` as it was before?

Reply:
Yep, thanks! This was a leftover change from when the `common/index` referenced code that could never run on the server since it relied on `window`. That was since moved into the `public` directory, but I never updated these references.